spec/factories/user_emails.rb

```diff
@@ -7,9 +7,14 @@
   factory :user_email, class: User::Email.name do
     primary true
     email
+    confirmed_at { Time.zone.now }
 
     after(:build) do |user_email|
       user_email.user ||= build(:user, emails: [user_email], emails_count: 0)
     end
+
+    trait :unconfirmed do
```

Comment:
I think traits are better positive, rather than negative.

So the default should be unconfirmed, the trait should be confirmed...

Unless the user emails are always confirmed when we use factory girl?